dist/bin/current/roddy.sh

```diff
@@ -129,7 +126,23 @@ elif [[ "$parm1" == "createworkflow" ]]; then
 fi
 
 IFS=""
-#[[ $RMIPORT != "" ]] && export DBG_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005" && echo "Opened up rmi debugger port"
-java $DBG_OPTS -cp .:$libraries:${RODDY_BINARY} de.dkfz.roddy.Roddy $*
-# -verbose:class
+export RGROOVY_LIB=`readlink -f ${RODDY_BINARY_DIR}/lib/groovy*.jar`
+
+debuggerSettings=""
+[[ -n ${DEBUG_ROODY} ]] && debuggerSettings=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+
+caller=$(checkAndDownloadGroovyServ "${RODDY_DIRECTORY}")
+
+echo "RGL: "$RGROOVY_LIB
+
+if [[ ${caller} == java ]]; then
+  echo "Using Java to start Roddy"
+	${caller} ${debuggerSettings} -cp .:$libraries:${RODDY_BINARY} de.dkfz.roddy.Roddy $*
```

Comment:
JAVA_OPTS are automatically used by groovyserver. Just checked it, it's hidden in the .go source files.